src/components/Terminal/index.tsx

```diff
@@ -108,7 +113,15 @@ class Terminal extends React.Component<ThemedTerminalProps, {}> {
 	}
 
 	public resize() {
-		this.fitAddon.fit();
+		const mountElementHasHeight =
+			this.mountElement &&
+			parseInt(
```

Comment:
Why not use `clientHeight` or `scrollHeight` instead?

Reply:
I prefer to use the same data used by `fitAddon` to calculate the columns, since if this data is missing the `fit()` function throw an error.
here the fitAddon function that calculate the element height:
<img width="1100" alt="Screenshot 2020-07-16 at 12 06 31" src="https://user-images.githubusercontent.com/7238159/87658718-de606780-c75c-11ea-8792-0aa485700633.png">